src/alerts/check_signals_updates.R

```diff
@@ -0,0 +1,207 @@
+box::use(jsonlite[fromJSON, toJSON])
+box::use(httr[GET, POST, add_headers, status_code, content])
+box::use(purrr)
+box::use(logger[log_error, log_info])
+
+box::use(../utils/get_env[get_env])
+box::use(../utils/hs_logger)
+box::use(cs = ../utils/cloud_storage)
+
+hs_logger$configure_logger()
+
+
+#' Queries the GitHub API to get all runs from a given monitoring workflow
+#' Requires workflows to be named following this convention: `monitor_<indicator_id>.yaml`
+#'
+#' NOTE: More data will be returned as each workflow is run more times. Perhaps won't scale well,
+#' depending on how many runs a workflow will have
+#'
+#' @param indicator_id ID of the indicator
+#'
+#' @returns JSON API response
+gh_status <- function(indicator_id) {
+  # TODO Temp cleaning before workflows are renamed to match indicator ids
+  ind <- sub("^[^_]*_", "", indicator_id)
+  workflow_id <- paste0("monitor_", ind, ".yaml")
+  token <- get_env("GH_TOKEN")
+  url <- paste0("https://api.github.com/repos/ocha-dap/hdx-signals/actions/workflows/", workflow_id, "/runs")
```

Comment:
So, it would be something like this in `{httr2}`:

```r
httr2$request(
   "https://api.github.com/repos/ocha-dap/hdx-signals/actions/workflows"
  ) |>
  httr2$url_path_append(
    workflow_id,
    "runs"
  ) |>
  httr2$req_auth_bearer_token(
    token = get_env("GH_TOKEN")
  ) |>
  httr2$req_perform() # can use httr2$req_method("GET") if not defaulting to GET, although I think it should
```